tests/acceptance/new-test.js

```diff
@@ -379,16 +379,56 @@ describe('Acceptance: ember new', function () {
       expect(file('.eslintrc.js')).to.equal(file(path.join(__dirname, '../fixtures', fixturePath, '.eslintrc.js')));
     }
 
-    function checkPackageJson(fixtureName) {
+    function checkFileWithEmberCLIVersionReplacement(fixtureName, fileName) {
       let currentVersion = require('../../package').version;
-      let fixturePath = path.join(__dirname, '../fixtures', fixtureName, 'package.json');
+      let fixturePath = path.join(__dirname, '../fixtures', fixtureName, fileName);
       let fixtureContents = fs
         .readFileSync(fixturePath, { encoding: 'utf-8' })
         .replace('<%= emberCLIVersion %>', currentVersion);
 
-      expect(file('package.json')).to.equal(fixtureContents);
+      expect(file(fileName)).to.equal(fixtureContents);
     }
 
+    it('app defaults', async function () {
```

Comment:
I created this because we actually didn't have any of our blueprint templating tests for `ember new foo` (we have tests for a two different permutations `--yarn` and `--no-welcome`).